lib/plugins/docker/index.js

```diff
@@ -4,13 +4,25 @@ import preset from '../../pattern-preset';
 
 export default class Docker {
 
+  static clickHandler({ image }) {
+    let isOffical = true;
+    const imageName = image.split(':')[0];
+
+    if (image.includes('/')) {
+      isOffical = false;
+    }
+
+    return [
+      `https://hub.docker.com/${isOffical ? '_' : 'r'}/${imageName}`,
+    ];
+  }
+
   getPattern() {
     return preset('Docker');
   }
 
   parseBlob(blob) {
-    insertLink(blob.el, DOCKER_FROM, {
-      resolver: 'dockerImage',
+    insertLink(this, blob.el, DOCKER_FROM, {
```

Comment:
Instead of passing the plugin as the first argument to `insertLink`, what if we just put it into the object literal, alongside `image`:

```js
insertLink(blob.el, DOCKER_FROM, {
  pluginName: this.constructor.name,
  image: '$1',
});
```

This way, `insertLink` won't have to pass an extra argument to `replace` and `buildDataAttr` at all. Note that doing this would invalidate my comment at https://github.com/OctoLinker/browser-extension/pull/274/files#r99657840, since the extra argument will no longer be present.